packages/veritone-react-common/src/components/TranscriptEngineOutput/TranscriptContent/index.js

```diff
@@ -172,21 +173,21 @@ export default class TranscriptContent extends Component {
               ? (selectedWord = words[0].word)
               : (selectedWord = '');
 
-            //---Update Snippet Data---
-            snippetSentences = snippetSentences + selectedWord + ' ';
-            snippetParts.push({
+            const snippet = {
               startTimeMs: entry.startTimeMs,
               stopTimeMs: entry.stopTimeMs,
               value: selectedWord
-            });
+            };
+            if (entry.guid) {
+              snippet.guid = entry.guid;
+            }
 
+            //---Update Snippet Data---
+            snippetSentences = snippetSentences + selectedWord + ' ';
```

Comment:
i'd like to see a lot of the logic in this function cleaned up, it's really messy right now. i.e

`
          //---Update Start & Stop Time---
          (snippetStartTime === undefined ||
            snippetStartTime > entry.startTimeMs) &&
            (snippetStartTime = entry.startTimeMs);
          (snippetStopTime === undefined ||
            snippetStopTime < entry.stopTimeMs) &&
            (snippetStopTime = entry.stopTimeMs);
`

what's wrong with `if/else`?

Reply:
Converted this to if then else.